docs/_data/navigation.yml

```diff
@@ -1,8 +1,6 @@
 pages:
   - title: What is Obscuro?
     children:
-      - title: Privacy Matters
-        url: what-is-obscuro/privacy-matters.html
       - title: Technology
```

Comment:
Seems odd to have a privacy L2 and yet not have a page which focuses on that at all on the docsite. I'd rather see this rewritten than removed entirely.

Reply:
I think its ok to repeat the homepage (you mean obscu.ro?) because some people will just go straight to the docsite knowing the website is not where the deeper tech content is

Reply:
I think that's the point though, people are going to the doc site to find the deeper tech not learn about "why privacy matters". What I can do instead is talk about privacy use-cases but the existing stuff was both wrong and marketing focused

Reply:
@GavT I've readded the section. Can you have another look, please